seqr/models.py

```diff
@@ -740,10 +728,45 @@ def __unicode__(self):
     class Meta:
        json_fields = [
            'guid', 'created_date', 'sample_type', 'dataset_type', 'sample_id', 'is_active', 'loaded_date',
-           'elasticsearch_index',
```

Comment:
this was only being used on the client side to determine whether or not a sample could be used for search, which is no longer necessary and was also kind of hacky to begin with